Makefile

```diff
@@ -25,41 +35,51 @@ all:
 	make size
 
 typescript:
-	cd typescript; npm i; time npm run compile;
+	cd typescript; npm i;
+	cd typescript; time $(tsc) && $(browserify) -p bundle-collapser/plugin ./dist/*.js | $(uglifyjs) --compress --mangle --screw-ie8 - > ../src/dist/bundle.js
```

Comment:
does this one work? I am impressed that it is able to resolve `bundle-collapser/plugin ` without node_modules